source/postprocess/visualization.cc

```diff
@@ -802,6 +797,14 @@ namespace aspect
                              "and a factor of 8 in 3d, when using quadratic elements for the velocity, "
                              "and correspondingly more for even higher order elements.");
 
+          prm.declare_entry ("Filter output", "false",
+                             Patterns::Bool(),
+                             " deal.II offers the possibility to filter duplicate vertices for hdf5 "
+                             " output files. This merges the vertices of adjacent cells and "
+                             " therefore saves disk space, but misrepresents discontinuous "
```

Comment:
You can omit the spaces at the beginning of each line since you already have them at the end.